### PR TITLE
deps: loosen the requirement constraints on django and python

### PR DESCRIPTION
**Pull request by gagantrivedi**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -42,8 +42,8 @@ include = [{ path = "migrations/sql/*", format = ["sdist", "wheel"] }]
 packages = [{ include = "task_processor"}]
 
 [tool.poetry.dependencies]
-python = "^3.10"
-django = "^3.2.23"
+python = ">=3.10,<4.0"
+django = ">=3.2.23"
```
Comment (matthewelwell) [inline pyproject.toml]: Just wondering if this should at least be <5?

Comment (gagantrivedi) [reply pyproject.toml]: Probably not worth doing in my opinion?

Comment (matthewelwell) [reply pyproject.toml]: I think we can go without for now, but we might see unit test failures in the future if there are breaking changes added to future major django versions.

Comment (gagantrivedi) [reply pyproject.toml]: The version is still locked, though, right? Unless someone updates the lock file, nothing should break?

Comment (matthewelwell) [reply pyproject.toml]: Sorry, yes, that's true 👍

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**